app/views/shared/_manage_volunteers.html.erb

```diff
@@ -0,0 +1,54 @@
+<h1 class="pt-5">Manage Volunteers</h1>
+
+<div id="volunteer-assignment" class="card card-container">
+  <div class="card-body">
+    <% if show_assigned_volunteers %>
+      <%= yield :table_title %>
+      <div class="table-responsive">
+        <table class='table case-list'>
```

Comment:
unfortunately, `case-list` also doesn't seem to do anything...but it's tacked onto tables all over the codebase. I left it here for now because there's a spec that's relying on it, but open to suggestions.

Reply:
Hmm that's odd. Let's file a ticket for it, it's probably an oops. https://github.com/rubyforgood/casa/issues/4006